src/components/Reports/FourteenMonthReports/Layout/Table/Table.tsx

```diff
@@ -0,0 +1,130 @@
+import React, { FC, forwardRef } from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  Box,
+  styled,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableRow,
+  Theme,
+  Typography,
+} from '@material-ui/core';
+import { Info as InfoIcon } from '@material-ui/icons';
+import {
+  FourteenMonthReportTableHead as TableHead,
+  FourteenMonthReportTableHeadProps as TableHeadProps,
+} from './TableHead/TableHead';
+import type { Contact, Month } from './TableHead/TableHead';
+
+interface FourteenMonthReportTableProps extends TableHeadProps {
+  isExpanded: boolean;
+  orderedContacts: Contact[] | undefined;
+  ref: React.Ref<HTMLTableElement>;
+}
+
+const NameTypography = styled(({ expanded: _expanded, ...props }) => (
+  <Typography {...props} />
+))(({ theme, expanded }: { theme: Theme; expanded: boolean }) => ({
+  marginLeft: expanded ? 0 : theme.spacing(1),
+}));
+
+const PrintableContainer = styled(TableContainer)(() => ({
+  // First style set size as landscape
+  // ['@media print']: {
+  //   ['@page']: { size: 'landscape' },
+  // },
```

Comment:
This style still relevant?

Reply:
Yeah, making `landscape` as the default print option would be good, but it is not working.